### ISSUE TITLE
Support thermostat devices with either `measure_temperature` or `target_temperature` but not both?

### ISSUE THREAD

**Issue by robertklep** [swttt/com.swttt.homekit#133]
For example, the [Danfoss radiator thermostats](https://apps.athom.com/app/com.danfoss) only define `target_temperature`.

**Comment by robertklep** [comment]
Yes, there is a workaround :)

From what I can see from Sprut's app, the currently _measured_ temperature would always be 0 (or at least fixed) in HomeKit for your Danfoss devices, is that correct?

**Comment by bonq** [comment]
As I said I swapped to yours so I can’t check but there was only one value as far as I remember. There where a setting for “heat” “cold” “automatic” but I never understood what that was about.

**Comment by robertklep** [comment]
Well, that's part of the issue: Homekit has a fixed set of properties for a thermostat device, one of those being "thermostat mode" (heating, cooling, etc). These properties are _always_ shown on iOS, even if they are not defined for the Homey device.

The same happens for the measured temperature: the Danfoss devices created in Homey don't have that property (they only have the _target_ temperature property), but iOS will still always show it.

Homeykit will, at the moment, not create a Homekit device from a Homey device if the latter misses certain properties. For thermostat-devices, it's the "measured temperature" property. And because your Danfoss devices don't have that property, they aren't shown on iOS.

So we can work around that, but we have to think of a way to not make it too confusing.

**Comment by bonq** [comment]
Aha, but I can remember that it showed more than 1 temperature. Maybe just set it to eg 0 then it should be rather obvious it’s nothing to care about?

**Comment by robertklep** [comment]
Yeah that, or perhaps set the measured temperature to the target temperature (although that might even be more confusing than just using "0" as default).

**Comment by Sinled** [comment]
Any news on this? If you don't have time i can add support with measured temp === target temp, i think it's better then 0, if we use 0 it will give strange temp summary in ios14 homekit e.g. 0 - 22 º
